### PR TITLE
**** Do not merge **** Common branch for testing Personal and non personal SOA CA DA

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### Change description ###

**** Do not merge **** Common branch for testing Personal and non personal SOA CA DA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java [hunk 1]
```diff
@@ -113,46 +93,6 @@ public EmailNotificationDetails sendEmailNotificationToCafcass(CaseData caseData
             .timeStamp(currentDate).build();
     }
 
-    private EmailTemplateVars buildApplicantSolicitorEmail(CaseData caseData, String solicitorName)
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java:116]: Why is this removed? is it related to SOA at all?

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java [hunk 2]
```diff
@@ -113,46 +93,6 @@ public EmailNotificationDetails sendEmailNotificationToCafcass(CaseData caseData
...
-            .caseReference(String.valueOf(caseData.getId()))
-            .caseName(caseData.getApplicantCaseName())
-            .solicitorName(solicitorName)
-            .caseLink(manageCaseUrl + URL_STRING + caseData.getId())
-            .privacyNoticeLink(privacy)
-            .issueDate(caseData.getIssueDate())
-            .build();
-    }
-
-    private EmailTemplateVars buildRespondentSolicitorEmail(CaseData caseData, String solicitorName,
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java:135]: Why is this removed? is it related to SOA at all?

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java [hunk 3]
```diff
@@ -420,17 +483,24 @@ private String handleNotificationsCaSolicitorCreatedCase(CaseData caseData, Stri
...
+                List<Document> packHiDocs = new ArrayList<>();
+                caseData.getRespondents().forEach(respondent -> packHiDocs.add(generateAccessCodeLetter(authorization, caseData,
+                                                                                                        respondent,
+                                                                                                        null,
+                                                                                                        PRL_LET_ENG_C100_RE6))
+                );
+                packHiDocs.addAll(getNotificationPack(caseData, PrlAppsConstants.HI, c100StaticDocs));
                 Map<String, Object> dynamicData = EmailUtils.getCommonSendgridDynamicTemplateData(caseData);
                 dynamicData.put("name", caseData.getApplicants().get(0).getValue().getRepresentativeFullName());
+                dynamicData.put("c100", true);
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java:496]: why is this field?

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java [hunk 4]
```diff
@@ -1344,48 +1596,58 @@ private List<Document> generatePackR(CaseData caseData, List<Document> staticDoc
...
 
     private List<Document> generatePackC(CaseData caseData, List<Document> staticDocs) {
         List<Document> docs = new ArrayList<>();
         docs.addAll(getCaseDocs(caseData));
-        docs.addAll(getDocumentsUploadedInServiceOfApplication(caseData));
-        docs.addAll(getNonC6aOrders(getSoaSelectedOrders(caseData)));
+        docs.addAll(getWitnessStatement(caseData));
         docs.addAll(staticDocs.stream()
-                        .filter(d -> !d.getDocumentFileName().equalsIgnoreCase(SOA_FL415_FILENAME)).toList());
+                                    .filter(d -> !d.getDocumentFileName().equalsIgnoreCase(SOA_FL415_FILENAME)).toList());
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java:1627]: Pack C and Pack D requires Fl415?

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java (hunk 1 above)
Comment (BharadwajSairam) [reply src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java:116]: Yes for gov notify we were using it so removed it since no longer required

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java (hunk 2 above)
Comment (BharadwajSairam) [reply src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java:135]: Yes for gov notify we were using it so removed it since no longer required

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java (hunk 3 above)
Comment (BharadwajSairam) [reply src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java:496]: for c100 and fl401 there is a slight change in text between templates, so using same template with if condition for text change
